src/simple-api/src/api/xxAMIDOxx.xxSTACKSxx.API.UnitTests/xxAMIDOxx.xxSTACKSxx.API.UnitTests.csproj

```diff
@@ -22,7 +22,7 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="coverlet.msbuild" Version="6.0.2">
+        <PackageReference Include="coverlet.msbuild" Version="3.2.0">
```

Comment:
Is there a reason for downgrading the coverlet?

Reply:
It broke the unit test coverage report. Coverlet was upgraded in the CQRS repo but that wasn't running the tests so it didn't cause the pipeline to fail and we only discovered it when we consolidated. Will look into the error but the priority was getting the pipelines working for the consolidated repo

Reply:
Yeah that makes sense 👍🏻